src/components/Layout/index.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react'
+import Helmet from 'react-helmet'
+import { ThemeProvider } from 'styled-components'
+// import config from '../../../data/SiteConfig'
+import './css/index.css'
+import './css/prism-okaidia.css'
+import { StaticQuery, graphql } from 'gatsby'
+import theme from './theme'
+
+// export default class MainLayout extends React.Component {
+//   getLocalTitle() {
+//     function capitalize(string) {
+//       return string.charAt(0).toUpperCase() + string.slice(1)
+//     }
+//     const pathPrefix = config.pathPrefix ? config.pathPrefix : '/'
+//     const currentPath = this.props.location.pathname
+//       .replace(pathPrefix, '')
+//       .replace('/', '')
+//     let title = ''
+//     if (currentPath === '') {
+//       title = 'Home'
+//     } else if (currentPath === 'tags/') {
+//       title = 'Tags'
+//     } else if (currentPath === 'categories/') {
+//       title = 'Categories'
+//     } else if (currentPath === 'about/') {
+//       title = 'About'
+//     } else if (currentPath.includes('posts')) {
+//       title = 'Article'
+//     } else if (currentPath.includes('tags/')) {
+//       const tag = currentPath
+//         .replace('tags/', '')
+//         .replace('/', '')
+//         .replace('-', ' ')
+//       title = `Tagged in ${capitalize(tag)}`
+//     } else if (currentPath.includes('categories/')) {
+//       const category = currentPath
+//         .replace('categories/', '')
+//         .replace('/', '')
+//         .replace('-', ' ')
+//       title = `${capitalize(category)}`
+//     }
+//     return title
+//   }
```

Comment:
Why is this code commented out? Please either remove it completely or leave it in. What is the background on this?

Reply:
I'm sorry. I hadn't certain about this changes.